src/tests/suite/launcher.sh

```diff
@@ -40,7 +40,7 @@ function kill_servers2 {
     do
 	tokens=( $hostline )
 	echo "killing " ${tokens[0]}
-	ssh -n ${tokens[0]} "pkill daos_io_server && pkill daos_server"
+	ssh -n ${tokens[0]} "pkill daos_engine && pkill daos_server"
```

Comment:
(lint) Add < /dev/null to prevent ssh from swallowing stdin. [SC2095]
(lint) In POSIX sh, array references are not supported. [SC2039]
(lint) Double quote to prevent globbing and word splitting. [SC2086]